src/spec.c

```diff
@@ -2176,6 +2213,42 @@ static void Indexes_ScanProc(RedisModuleCtx *ctx, RedisModuleString *keyname, Re
   ++scanner->scannedKeys;
 }
 
+
+static void DebugIndexes_ScanProc(RedisModuleCtx *ctx, RedisModuleString *keyname, RedisModuleKey *key,
```

Comment:
Please move all the debug functions to a separate file, or at least to the end of this file with a clear separation